### DISCUSSION TITLE
What does the PDM acronym stand for?

### DISCUSSION THREAD

**Discussion by Macktireh** [pdm-project/pdm#3357] [Q&A]
Could you please tell me what the acronym PDM stands?

**Comment by frostming** [comment]
Look at the banner of https://pdm-project.org and pick one you like.

**Reply by Macktireh** [reply]
That's why I'm asking the question to get the official acronym, not based on my own preference.

**Reply by frostming** [reply]
No official one. We don't want to limit the scope of PDM to an official full name, though it's basically a Python package manager.

**Comment by ncoghlan** [comment]
I must admit I thought it just stood for "Python Development Manager", but now I've been alerted to the everchanging initialism header, so I'm calling that a win :)
